dnf-behave-tests/features/module-platform.feature

```diff
@@ -93,13 +88,7 @@ Scenario: I can't update pseudo-module
 Scenario: I can't remove pseudo-module
  When I execute dnf with args "module remove pseudoplatform:6.0"
  Then the exit code is 0
-  And stdout matches line by line
-  """
-  ?Last metadata
-  ^Dependencies resolved.
-  ^Nothing to do.
-  ^Complete!
-  """
+  And stdout contains "Nothing to do."
```

Comment:
Same here, check for exact match if possible.

Reply:
Thanks, both are fixed now.